.github/workflows/broken-link-check-schedule.yml

```
@@ -0,0 +1,37 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+# Based on hashicorp/consul/.github/workflows/broken-link-check.yml
+# Run lychee link check every Sat at 2am
+# Create GH issue from lychee output; assign issue to Aimee Ukasick
+
+name: Broken Link Check Scheduled
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 2 * * SAT" # every Sat at 2 am
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+
+      - name: Run lychee link checker
+        id: lychee
+        uses: lycheeverse/lychee-action@2b973e86fc7b1f6b36a93795fe2c9c6ae1118621 # v1.10.0
+        with:
+          args: ./website/content/docs/ --base https://developer.hashicorp.com/ --exclude-all-private --exclude '\.(svg|gif|jpg|png)' --exclude 'manage\.auth0\.com' --accept 403 --max-concurrency=24 --no-progress --verbose
+          # Fail GitHub action when broken links are found?
+          fail: false
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create GitHub Issue From lychee output file
+        if: env.lychee_exit_code != 0
+        uses: peter-evans/create-issue-from-file@24452a72d85239eacf1468b0f1982a9f3fec4c94 # v5.0.0
+        with:
+          title: Link Checker Report
+          content-filepath: ./lychee/out.md
+          labels: automated, theme/docs
+          assignees: aimeeu
```

I think you need a write token?

```suggestion
          assignees: aimeeu
          
permissions:
  contents: read
  id-token: write          
```

We're also missing the trailing EOL here.